numasched/numasched.bt

```diff
@@ -0,0 +1,45 @@
+#!/bin/env bpftrace
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * numasched	Trace task switch numa
+ *		For Linux, uses bpftrace and eBPF.
+ *
+ * Also a basic example of bpftrace. This tool need bpftrace numaid builtin.
```

Comment:
The numaid builtin was added almost 2 years ago so I don't think we need to specify that. However, I added a bit about the kernel version requirement.

```suggestion
 * This tool requires a kernel version >= 5.8
```